webapp/components/MasonryGrid/MasonryGridItem.spec.js

```diff
@@ -10,23 +9,22 @@ describe('MasonryGridItem', () => {
   let wrapper
 
   beforeEach(() => {
-    wrapper = shallowMount(MasonryGridItem, { localVue })
-    wrapper.vm.$parent.$emit = jest.fn()
+    wrapper = mount(MasonryGridItem, { localVue })
   })
 
-  it('emits "calculating-item-height" when starting calculation', async () => {
+  it('parent emits "calculating-item-height" when starting calculation', async () => {
     wrapper.vm.calculateItemHeight()
     await wrapper.vm.$nextTick()
 
-    const firstCallArgument = wrapper.vm.$parent.$emit.mock.calls[0][0]
-    expect(firstCallArgument).toBe('calculating-item-height')
+    const firstEmittedFunction = wrapper.vm.$parent.__emittedByOrder[0]
```

Comment:
I told my students not to test against `wrapper.vm` unless there is no other way. Couldn't we test the `MasonryGrid` component and use `mount` then click on a `MasonryGridItem` component and check if the `wrapper.emitted()` contains `'calculating-item-height'`?